### PR TITLE
Docker File Modification

### PR DESCRIPTION
**Pull request by Aibaki**
Changed the Docker file to use the config env file

### CODE REVIEW

File: Analytics/add_startup_admin.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class AddStartupAdmin(Command):
     def __init__(self):
         """
         Create a local sqlalchemy engine that connects to the database
-        defined in settings.py
+        defined in settings.py.bak
```
Comment (WizardNo7) [inline Analytics/add_startup_admin.py]: Since settings.py files are no longer used, do we remove or update these comments?

Comment (Aibaki) [reply Analytics/add_startup_admin.py]: I think those aren't needed. I think it was suggested elsewhere.

File: Analytics/gunicornserver.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
 from flask_script import Command, Option
-
 from settings.get_config_decorator import GetConfig
+from typing import Union
+import os
 
+from settings.get_config_decorator import GetConfig
```
Comment (WizardNo7) [inline Analytics/gunicornserver.py]: This is a duplicate line

File: Analytics/settings.py.bak [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import os
```
Comment (WizardNo7) [inline Analytics/settings.py.bak]: This file is obsolete

File: Analytics/settings/config.env.yml [hunk 4]
```diff
@@ -0,0 +1,196 @@
+api_endpoints:
```
Comment (WizardNo7) [inline Analytics/settings/config.env.yml]: Don't check this in

Comment (WizardNo7) [reply Analytics/settings/config.env.yml]: Distribute externally

Comment (Aibaki) [reply Analytics/settings/config.env.yml]: Deleted the file check commits below

File: Dockerfile-UI [hunk 5]
```diff
@@ -1,12 +1,14 @@
...
 
-RUN mkdir /code
-WORKDIR /code
+RUN mkdir /Frontend /Analytics
+RUN mkdir /Analytics/settings
 # Install dependencies
 # package*.json used to copy files package.json and package-lock.json
-COPY Frontend ./
+COPY /Frontend /Frontend
+COPY /Analytics/settings/config.env.yml /Analytics/settings/config.env.yml
```
Comment (WizardNo7) [inline Dockerfile-UI]: This line will crash now

Comment (WizardNo7) [reply Dockerfile-UI]: Unless you give them explicit instructions in your readme/doc on adding the file in there

**Comment by WizardNo7** [comment]
Looks good

**Pull request merged**